fps/src/main/scala/chapter03/exercise3_23.scala

```diff
@@ -0,0 +1,13 @@
+
+package chapter03
+
+import chapter03.listImplementation.*
+import chapter03.listImplementation.List.*
+
+import scala.annotation.tailrec
+
+
+object exercise3_23 extends App:
+
+ 
+  println(map(zip(List("a","b","c"), List("d","e","f")))((x,y) => x + y))
```

Comment:
same for `map(map(zip(List("a","b","c"), List("d","e","f")))(_+_)

Reply:
no - you need to do: (_+_)

with the underscores meaning that you pass the first element from the tuple after zip on the left and the second on the right of the plus.

( the underscores in the previous example were interpreted as italics by github - my bad 8))